app/assets/stylesheets/themes/cultural_repository.scss

```diff
@@ -1,53 +1,68 @@
-.cultural_repository  {
+.cultural_repository {
   ////// Navbar //////
   .clear-right {
     clear: right !important;
   }
+
   .vertical-center-image img {
     vertical-align: middle;
     display: inline-block;
     height: 100%;
   }
+
   body.dashboard {
     padding-top: 100px !important;
     @media (max-width: $screen-md-min) {
       padding-top: 0px !important;
     }
   }
+
   div.form-group > label.control-label.col-sm-4 {
     color: #dcdcdc;
     font-weight: 400 !important;
   }
+
   // Note: copied over to override /* line 555, /usr/local/rvm/gems/ruby-2.5.3/gems/bootstrap-sass-3.4.1/assets/stylesheets/bootstrap/_navbar.scss */
-  .navbar-inverse .navbar-nav > .active > a, .navbar-inverse .navbar-nav > .active > a:hover, .navbar-inverse .navbar-nav > .active > a:focus {
+  .navbar-inverse .navbar-nav > .active > a,
+  .navbar-inverse .navbar-nav > .active > a:hover,
+  .navbar-inverse .navbar-nav > .active > a:focus {
     background-color: transparent !important;
   }
+
   // Note: copied over to override /* line 5, /home/app/webapp/app/assets/stylesheets/hyku.scss */
   #masthead #logo img {
     max-height: 100px !important;
   }
+
   $navbar-height: 100px !important;
   ////// End Navbar //////
 
-  div#featured_works > div.featured-works-4-column-layout:nth-child(1n+5){ display: none !important; }
+  div#featured__works > div.featured-works-4-column-layout:nth-child(1n+5) {
```

Comment:
did you mean to change this to use the double underscore?

Reply:
There was a Block Element Modifier (BEM) error on this one that Dylan and I worked on to come up with a solution. We found that the format was requesting a double underscore to pass. Should I change it back to a single underscore?

Reply:
Yes that would be great Erik, thank you!

Reply:
Not a problem! Double underscore removed to a single underscore and the code has been pushed up for review.